src/main/java/org/commonjava/migrate/pathmap/CassandraMigrator.java

```diff
@@ -46,10 +57,32 @@
 
     private final ChecksumCalculator checksumCalculator;
 
+    private final PreparedStatement preparedStoresIncrement;
+
+    private final String keyspace = "indycache";
+
+    private final String gaStorePattern = "^build-\\d+";
```

Comment:
Not sure if this will be changed, but better way is using a command option to be passed by cli flag for this one.